tests/suite/symbols/symbol.typ

```diff
@@ -49,3 +49,8 @@
 --- symbol-unknown-modifier ---
 // Error: 13-20 unknown symbol modifier
 #emoji.face.garbage
+
+--- symbol-repr ---
+#repr(sym.amp) \
```

Comment:
Maybe it's better to use `test` to compare strings directly?

Reply:
Done